### ISSUE TITLE
Add description field to strategy params

### ISSUE THREAD

**Issue by ivarconr** [Unleash/unleash-frontend#2]
Strategy parameters only has name field, but a description field would add more help to the user when using a strategy. 

I purpose the new format for strategy definitions:

```json
{
    "name": "ActiveForUserWithEmail",
    "description": "A comma separated list of email adresses this feature should be active for.",
    "parametersTemplate": {
        "emails": "string"
    },
    "parametersDescription": {
        "emails": "Some optional field describing the field."
    }

}
```

**Comment by ivarconr** [comment]
Or maybe we should change the format?

```json
{
    "name": "ActiveForUserWithEmail",
    "description": "A comma separated list of email adresses this feature should be active for.",
    "parametersTemplate": {
        "emails":  {
             "type": "string",
             "description": "A comma seperated list of emails",
             "required": true
       }
    }
}
```

**Comment by ivarconr** [comment]
Will be solved as part of #14 and already added suport in the API (https://github.com/Unleash/unleash/pull/183)
